Frameworks/UIKit/UIActivityIndicatorView.mm

```
@@ -229,13 +242,12 @@ - (UIActivityIndicatorViewStyle)activityIndicatorViewStyle {
 */
 - (void)startAnimating {
     RunSynchronouslyOnMainThread(^{
-        if (_isAnimating) {
-            return;
+        if (!_isAnimating) {
+            _isAnimating = YES;
+            [self setHidden:NO];
```

Yes, because by default iOS hides the indicator when it is not animating.